src/gui/windows/main.zig

```diff
@@ -18,8 +22,135 @@ const padding: f32 = 8;
 fn exitGame(_:usize) void {
 	main.Window.c.glfwSetWindowShouldClose(main.Window.window,main.Window.c.GLFW_TRUE);
 }
+
+fn deleteDirectoryRecursive(fs: std.fs.Dir, allocator: std.mem.Allocator, path: []const u8) anyerror!void {
+	var dir = try fs.openDir(path, .{.iterate = true});
+	defer dir.close();
+
+	var it = dir.iterate();
+
+	while (try it.next()) |entry| {
+		if (entry.kind == .file) {
+			try dir.deleteFile(entry.name);
+		} else if (entry.kind == .directory) {
+			try deleteDirectoryRecursive(dir, allocator, entry.name);
+		}
+	}
+
+	try fs.deleteDir(path);
+}
+
+pub fn play(_:usize) void {
+	deleteDirectoryRecursive(std.fs.cwd(), std.heap.page_allocator, ".cubyz/Cubyz-master") catch |err| {
+		std.debug.print("{}\n", .{err});
```

Comment:
If you print error codes this is kind of unhelpful, since you only get the error name.

Instead I always add some extra text messages to each error, like `Error while deleting .cubyz/Cubyz-master directory: {}`

Additionally please use `std.log` so that it gets print to the log file, and maybe in the future we could even display the launcher log in the launcher itself.